Source/Scene/Cesium3DTileset.js

```diff
@@ -849,6 +855,41 @@ define([
         },
 
         /**
+         * The maximum amount of memory in MB that can be used by the tileset.
+         * Tiles not in view are unloaded to enforce this.
+         * <p>
+         * If decreasing this value results in unloading tiles, the tiles are unloaded the next frame.
+         * </p>
+         * <p>
```

Comment:
Add a paragraph describing what memory usage this is, e.g.,
* Vertices/indices
* Textures
* Batch Table? Probably 2x since this is in our CPU memory as well as the GPU textures